### ISSUE TITLE
More accessible exports

### ISSUE THREAD

**Issue by allan2** [Aiving/material-colors#13]
The current exports are organized in line with  [material-foundation/material-utilities](https://github.com/material-foundation/material-color-utilities).
Given the recent changes making the API more idiomatic to Rust, perhaps the exports could be restructured as well.

Looking at the list of exports:
```rust
pub use hct::Hct;
pub use scheme::content::SchemeContent;
pub use scheme::expressive::SchemeExpressive;
pub use scheme::fidelity::SchemeFidelity;
pub use scheme::fruit_salad::SchemeFruitSalad;
pub use scheme::monochrome::SchemeMonochrome;
pub use scheme::neutral::SchemeNeutral;
pub use scheme::rainbow::SchemeRainbow;
pub use scheme::tonal_spot::SchemeTonalSpot;
pub use scheme::vibrant::SchemeVibrant;
pub use scheme::Scheme;
pub use utils::color::Argb;
pub use utils::color::Rgb;
pub use utils::theme::theme_from_source_color;
pub use palettes::core::CorePalette;
pub use palettes::tonal::TonalPalette;
```

It looks like a lot of modules just house one thing.

Could it be reorganized to be easier for the API consumer?

Some suggestions:
- move `utils::color` to `color`
- move `utils::theme` to `Theme`
- rename `palettes` to `palette`
- group the  scheme variants somehow

```rust
pub use hct::Hct;
pub use color::{Argb, Rgb};
pub use palette::{CorePalette, TonalPalette};
pub use scheme::{Scheme, variant::{SchemeContent, SchemeExpressive, ...};
pub use theme::Theme;
```

**Comment by Aiving** [comment]
Okay, I'll fix it!

**Comment by Aiving** [comment]
closed in e50bb2e
